app/controllers/calls_controller.rb

```diff
@@ -50,9 +50,9 @@ def connection_gather_prompt
         action: calls_connection_gather_url(connection_id: active_connection.id),
         timeout: 15
       ) do |gather|
-        r.Play AudioFileFetcher.file_for_key('user_response')
+        r.Play AudioFileFetcher.audio_url_for_key('user_response')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.